src/api/app/helpers/webui/buildresult_helper.rb

```diff
@@ -28,4 +43,34 @@ def arch_repo_table_cell(repo, arch, package_name, status = nil, enable_help = t
       end
     end
   end
+
+  def arch_repo_table_cell_webui2(repo, arch, package_name, status = nil, enable_help = true)
```

Comment:
We need to kill this. Not necessarily in the PR. Can you add a card about it to the backlog?